src/components/GrantsInfographics/GrantsInfographics.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
+import RandomColor from 'randomcolor';
+import Chart from './Chart';
+import Bar from './Bar';
+import './scss/index.scss';
+import './scss/react-vis.scss';
+
+class GrantsInfograpics extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      issue: [],
+      location: [],
+      awarded: '',
+      projects: '',
+      barData: [],
+      chartData1: [],
+      chartData2: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch(this.props.grantsAPI)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error('something went wrong');
+        }
+        return response.json();
+      })
+      .then((response) => {
+        const grantData = response.data.facets;
+        const awardedTotal = this.sum(grantData.issue.map(a => a.total_awarded));
+        const projectTotal = this.sum(grantData.issue.map(a => a.count));
+        const barData = [];
+        const chartData1 = [];
+        const chartData2 = [];
+
+        grantData.issue.map((data) => {
+          barData.push({ x: data.key, y: data.count });
+          return barData;
+        });
+
+        grantData.country_name.filter((r) => { return r.key === 'UNITED KINGDOM'; }).map((data) => {
+          const projects = Math.trunc((data.count/projectTotal)*100);
```

Comment:
I think trunc all over the items mayblead to having total not equal to 100%, probably less